src/nlp.jl

```diff
@@ -321,10 +394,19 @@ function MOI.initialize(d::NLPEvaluator, requested_features::Vector{Symbol})
 
     num_variables_ = num_variables(d.m)
 
-    moi_index_to_consecutive_index = Dict(moi_index => consecutive_index for (consecutive_index, moi_index) in enumerate(MOI.get(d.m, MOI.ListOfVariableIndices())))
+    moi_index_to_consecutive_index = Dict(
+        moi_index => consecutive_index
+        for
+        (consecutive_index, moi_index) in
+        enumerate(MOI.get(d.m, MOI.ListOfVariableIndices()))
```

Comment:
This one is a bit weird. Not sure what is better though

Reply:
Issue: https://github.com/domluna/JuliaFormatter.jl/issues/314

I'm not opposed to merging as-is and having this fixed in a future update of the formatter.